portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,28 +32,45 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> messages;
+  private List<String> comments; //update to custom struct later
   
   @Override
   public void init() {
-      messages = new ArrayList<>();
-      messages.add("Peanuts aren\'t actually nuts");
-      messages.add("A group of squids is called a squad");
-      messages.add("The speed of a computer mouse is measured in \'Mickeys\'");
+      comments = new ArrayList<>();
   }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJson(messages);
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.ASCENDING);
```

Comment:
Maybe sort the results in descending order so that the most recent comments appear first?